.pre-commit-config.yaml

```diff
@@ -21,14 +21,14 @@ repos:
   hooks:
   - id: black
     files: ^bin/update_pythons.py$
-    args: ["--line-length=120"]
```

Comment:
Why is this being removed? Feels unrelated.

Reply:
Oh, nevermind. Moved down, I see now!

Reply:
It had to be manually given if running black directly. Now, it isn't required if running black directly.